doc/modules/linear_model.rst

```diff
@@ -147,7 +147,7 @@ as GridSearchCV except that it defaults to Generalized Cross-Validation
             normalize=False, scoring=None, store_cv_values=True)
     >>> reg.alpha_                                          # doctest: +SKIP
     1.0
-    >>> reg.cv_values_
+    >>> reg.cv_values_                                      # doctest: +SKIP
```

Comment:
You could use `# doctest: +NORMALIZE_WHITESPACE +ELLIPSIS`  instead and then use `...` after the third digit, e.g. `0.105...`.